dist/app.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+
```

Comment:
why did you use /dist folder? I think it should be in gitignore other you can explain how you are using it